### DISCUSSION TITLE
Carry a alias over an ssh session

### DISCUSSION THREAD

**Discussion by OmerBenisty** [kovidgoyal/kitty#7300] [Q&A]
Hello,
Im struggling to find away to carry aliases over ssh sessions, I see that when using `kitten ssh` its auto creating the alias `edit-in-kitty='kitten edit-in-kitty'` is there a way I can add an alias of my own using a similar manner?

Thanks.

**Comment by kovidgoyal** [comment]
The ssh kitten can copy over shell rc files as well, where aliases are
defined, see the docs
https://sw.kovidgoyal.net/kitty/kittens/ssh/#a-real-world-example

**Reply by OmerBenisty** [reply]
Do you have an example involving bash?
I'm trying to do it in a way im not overwriting the pre-existing bash config and couldn't manage to do that

**Comment by kovidgoyal** [comment]
No bash doesnt have the facilities for doing it that way automatically.

**Comment by sandman7920** [comment]
`.bashrc` on Ubuntu has
```bash
# Alias definitions.
# You may want to put all your additions into a separate file like
# ~/.bash_aliases, instead of adding them here directly.
# See /usr/share/doc/bash-doc/examples in the bash-doc package.

if [ -f ~/.bash_aliases ]; then
    . ~/.bash_aliases
f
```

**kitty** `ssh.conf`
```
hostname *
copy --dest .bash_aliases .bash_aliases_kitty
```
